tensorrt_llm/_torch/modules/fla/fused_sigmoid_gating_recurrent.py

```
@@ -46,117 +52,127 @@ def fused_sigmoid_gating_delta_rule_update_kernel(
-        )
-        b_g = -tl.exp(b_A_log) * softplus_x
+    while i_nh < total_nh:
+        i_n, i_hv = i_nh // HV, i_nh % HV
+        i_h = i_hv // (HV // H)
+
+        if IS_VARLEN:
+            bos, eos = (
+                tl.load(cu_seqlens + i_n).to(tl.int64),
+                tl.load(cu_seqlens + i_n + 1).to(tl.int64),
+            )
+            all = T
```

Ruff (0.15.7)

[error] Variable `all` is shadowing a Python builtin

(A001)